src/EPR.Calculator.API.UnitTests/CreateDefaultParameterDataValidatorTest.cs

```diff
@@ -114,7 +114,8 @@ public void ValidateTest_For_Missing_Unique_References()
             var dto = new CreateDefaultParameterSettingDto
             {
                 ParameterYear = "2024-25",
```

Comment:
Do you want to add a Unit test for 
No File name being sent
Max Size 
Empty String

Reply:
Resolved comments